app/Agri.Models/Farm/FarmDetails.cs

```diff
@@ -17,5 +17,10 @@ public FarmDetails()
         public bool HasAnimals { get; set; }
         public bool ImportsManureCompost { get; set; }
         public bool UsesFertilizer { get; set; }
+        public bool HasAnimals1 { get; set; }
```

Comment:
@NavathaMundru2019 Please add a comment above the HasAnimals1 property

//TODO: Remove when Menu has been revised